pycon/templates/finaid/receipt_form.html

```diff
@@ -0,0 +1,65 @@
+{% extends "site_base.html" %}
+
+{% load bootstrap_tags %}
+{% load compress %}
+{% load static %}
+{% load i18n %}
+{% load boxes_tags %}
+
+{% block head_title %}Receipt Form{% endblock %}
+
+{% block body_id %}receipt_form{% endblock %}
+
+{% block page_title %}Receipt Application{% endblock %}
+
+{% block body %}
+    {% if form.errors %}
+        <div class="alert">
+            <p class="text-error">
+                There were errors in your form, please correct them and submit again.
+            </p>
+        </div>
+    {% endif %}
+
+    <form method="POST" action="" enctype="multipart/form-data" class="form-horizontal">
+        {% csrf_token %}
+        {% if form.errors and not form.non_field_errors %}
+          <div class="alert alert-error">
+            {% blocktrans count counter=form.errors.items|length %}Please correct the error below.{% plural %}Please correct the errors below.{% endblocktrans %}
```

Comment:
This seems to work from trying it out, and is very similar to the example in [this](https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#std:templatetag-blocktrans) doc page, namely:

    {% blocktrans count counter=list|length %}
    There is only one {{ name }} object.
    {% plural %}
    There are {{ counter }} {{ name }} objects.
    {% endblocktrans %}